tasks/MatchedCatalogsAggregation.py

```diff
@@ -43,35 +39,9 @@ class MatchedCatalogsAggregationTask(pipeBase.PipelineTask):
     ConfigClass = MatchedCatalogsAggregationTaskConfig
     _DefaultName = "matchedCatalogsAggregationTask"
 
-#    def __init__(self, config: pipeBase.PipelineTaskConfig, *args, **kwargs):
-#        super().__init__(*args, config=config, **kwargs)
-
     # Accepts python objects
     def run(self, measurements):
         self.log.info(f"Computing the mean of PA1 values in matched catalogs")
-        
-        print(measurements)
-        
-        measurement_array = u.Quantity([x.quantity for x in measurements], u.mmag)
-        print(measurement_array)
-        print(np.mean(measurement_array))
-        
-        #measurement_array = []
-        #for ii in range(0, len(measurements)):
-        #    print(ii, measurements[ii])
-        #    measurement_array.append()
-        #meas = np.array(0.)
-        #meas = Measurement("numSources", nSources * u.count)
-        input('WAIT')
-        return pipeBase.Struct(summary=Measurement("MeanPA1", np.mean(measurement_array)))
 
-    # Use this to assemble python objects to send to the run method
-    def runQuantum(self, butlerQC,
-                   inputRefs,
-                   outputRefs):
-        inputs = butlerQC.get(inputRefs)
-        outputs = self.run(**inputs)
-        butlerQC.put(outputs, outputRefs)
-#        inputs['vIds'] = [el.dataId.byName() for el in inputRefs.__dict__['source_catalogs']]
-#        outputs = self.run(**inputs)
-#        butlerQC.put(outputs, outputRefs)
\ No newline at end of file
+        value = np.mean(u.Quantity([x.quantity for x in measurements if np.isfinite(x.quantity)]))
+        return pipeBase.Struct(summary=Measurement("MeanPA1", value))
```

Comment:
Should there be a new line here?

Reply:
I think so, yes.  Jeff also made some comments on slack, I'll incorporate those.